packages/react-dev-overlay/src/internal/container/Errors.tsx

```diff
@@ -161,23 +159,10 @@ export const Errors: React.FC<ErrorsProps> = function Errors({ errors }) {
   React.useEffect(() => {
     if (errors.length < 1) {
       setLookups({})
-      setMinimized(false)
       setActiveIndex(0)
     }
   }, [errors.length])
 
-  const minimize = React.useCallback((e?: MouseEvent | TouchEvent) => {
```

Comment:
I don't think this should be removed 🤔

Reply:
@timneutkens This is never used
![code](https://i.imgur.com/MFbucSN.png)